src/services/config-service.ts

```diff
@@ -0,0 +1,73 @@
+import * as fs from 'node:fs'
+import * as path from 'node:path'
+import * as os from 'node:os'
+import { CLIError } from '@oclif/core/lib/errors'
+import { CONFIG_FOLDER, CONFIG_FILE } from '../paths'
+
+export default class ConfigService {
+  private _config: any
```

Comment:
If we know that the format is always a key/value map of strings we may replace that any with something more specific:

    private _config: {[key:string]: string} = {}

Reply:
@zonia3000 thanks for the review. I'm not sure about that, @cecchisandrone WDYT?

Reply:
yes, for the moment it's a reasonable choice

Reply:
Done